yt/frontends/ramses/data_structures.py

```diff
@@ -630,7 +635,13 @@ def read_rhs(cast):
         # This is likely not true, but it's not clear how to determine the boundary conditions
         self.periodicity = (True, True, True)
         # These conditions seem to always be true for non-cosmological datasets
-        if rheader["time"] >= 0 and rheader["H0"] == 1 and rheader["aexp"] == 1:
+        if self.force_cosmological is not None:
+            is_cosmological = self.force_cosmological
+        else:
+            is_cosmological = (rheader["time"] >= 0 and
+                               rheader["H0"] == 1 and
+                               rheader["aexp"] == 1)
```

Comment:
There's a logic error here. Before we had if these conditions are true, then it's *not* a cosmology simulation. But here you're saying it's a cosmology simulation if these conditions are true.

I *think* the fix is just to change this to a `not` in front of the boolean expression